config/locales/en/providers.yml

```diff
@@ -765,9 +765,10 @@ en:
         no_button: No, do not delete it
     has_evidence_of_benefits:
       show:
-        page_title: "Do you have evidence that your client receives %{passporting_benefit}?"
+        h1_no_partner: "Do you have evidence that your client receives %{passporting_benefit}?"
+        h1_partner: "Do you have evidence that your client is covered by their partner's %{passporting_benefit}?"
         hint: "For example, a letter from the DWP. A caseworker will request this after you've submitted your application."
-        evidence_hint: 'For example, a letter from the DWP, a bank statement from the last 3 months showing benefits paid in, or any other relevant documents.'
+        evidence_hint: 'For example, a letter from DWP or any other relevant documents.'
```

Comment:
Is this a duplicate of 770? Do we need both?
Also, it may be worth checking with Louise on the wording. Should we use an `abbr` tag on DWP, and I've sounds wrong without a `the` prefix in this sentence.
You wouldn't say `bring me a letter from Department of Work and Pensions` you'd say `*the* Department of Work and Pensions`

Reply:
Have checked this with Louise and she says it's the right content 👍 apparently you don't need 'the' for acronyms - this is according to the style guide

Reply:
https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style